### PR TITLE
[Move Expando]LPS-192380 Use createExpandoValue(long valueId) instead of using new …

### PR DESCRIPTION
**Pull request by slnn**
https://liferay.atlassian.net/browse/LPS-192380

### CODE REVIEW

**Comment by catarineaguiar** [comment]
no regressions found

**Comment by slnn** [comment]
ci:test:sf

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
ci:test:search

File: modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/parameter/contributor/UserSXPParameterContributor.java [hunk 1]
```diff
@@ -263,7 +263,8 @@ private void _addExpandoSXPParameters(
 				expandoColumn.getColumnId());
 
 			if (expandoValue == null) {
-				expandoValue = new ExpandoValueImpl();
+				expandoValue = ExpandoValueLocalServiceUtil.createExpandoValue(
```
[the remark this answers is not in the document]
Comment (slnn) [reply modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/parameter/contributor/UserSXPParameterContributor.java]: Done

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
@tinatian Hi Tina, could you priority this one? Thanks!

**Comment by shuyangzhou** [comment]
ci:forward:force

**Pull request closed without merging**